src/ui/layouts/runs/RunDetail/useService.ts

```diff
@@ -45,6 +48,31 @@ export const useService = (): ServiceInterface => {
     }
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [isMounted, setIsMounted]);
+  useEffect(() => {
+    if (run.status === 'running') {
+      const intervalId = setInterval(() => {
+        dispatch(
+          runsActions.runForId({
+            runId: runId,
+            onSuccess: (res) => {
+              if (res.status !== 'running') {
+                dispatch(
+                  runsActions.graphForRun({
+                    runId: runId,
+                  }),
+                );
+              }
+            },
+          }),
+        );
+      }, 5000);
```

Comment:
Some of these intervals differ from each other Is there a thought behind not making it consistent?